src/lib/git/azure.ts

```diff
@@ -366,11 +366,10 @@ export const repositoryHasFile = async (
   );
 
   if (gitItem === null) {
-    throw Error(
-      "Error installing build pipeline. Repository does not have a " +
-        fileName +
-        " file."
-    );
+    throw buildError(errorStatusCode.GIT_OPS_ERR, {
+      errorKey: "git-azure-file-no-exist-in-repo",
```

Comment:
can we just call this git-azure-file-exist? i don't see a reason why we'd throw an error _if_ the file exists

Reply:
```
  if (gitItem === null) {
    throw buildError(errorStatusCode.GIT_OPS_ERR, {
      errorKey: "git-azure-file-no-exist-in-repo",
      values: [fileName, branch, repoName],
    });
  }
```
if `gititem === null` means that file does not exist. Am I right? Or it is the other way around. `gititem === null` means file exist?